lib/shared/export-snippet.ts

```diff
@@ -51,6 +53,7 @@ const OUTPUT_EXTENSIONS: Record<ExportFormat, string> = {
   kicad_pcb: ".kicad_pcb",
   kicad_zip: "-kicad.zip",
   "kicad-footprint-library": "-footprints.zip",
+  "kicad-library": "-kicad-library.zip",
```

Comment:
I think i mentioned this before but lets not output zips, just output a directory